### PR TITLE
chore: introduce accessibility tab in recorder

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: packages/playwright-core/src/server/injected/consoleApi.ts [hunk 1]
```diff
@@ -85,7 +85,11 @@ class ConsoleAPI {
       inspect: (selector: string) => this._inspect(selector),
       selector: (element: Element) => this._selector(element),
       generateLocator: (element: Element, language?: Language) => this._generateLocator(element, language),
-      ariaSnapshot: (element?: Element) => this._injectedScript.ariaSnapshot(element || this._injectedScript.document.body),
+      ariaSnapshot: (element?: Element) => {
+        const snapshot = this._injectedScript.ariaSnapshot(element || this._injectedScript.document.body);
+        // eslint-disable-next-line no-console
+        console.log(snapshot);
```
Comment (mxschmitt) [inline packages/playwright-core/src/server/injected/consoleApi.ts:91]: maybe we can write it into their clipboard

File: utils/build/build.js [hunk 2]
```diff
@@ -283,7 +283,7 @@ for (const webPackage of ['html-reporter', 'recorder', 'trace-viewer']) {
       'vite',
       'build',
       ...(watchMode ? ['--watch', '--minify=false'] : []),
-      ...(withSourceMaps ? ['--sourcemap'] : []),
+      ...(withSourceMaps ? ['--sourcemap=inline'] : []),
```
Comment (mxschmitt) [inline utils/build/build.js:286]: Note: This is a drive-by that because of a recent DevTools regression which prevents source-maps from working correctly with single token domains.

File: packages/playwright-core/src/server/injected/consoleApi.ts (hunk 1 above)
Comment (yury-s) [inline packages/playwright-core/src/server/injected/consoleApi.ts:91]: We don't print to console in other methods, why not just return the value as before?

**Review by yury-s — APPROVED** [review]

File: packages/playwright-core/src/server/injected/consoleApi.ts (hunk 1 above)
Comment (pavelfeldman) [reply packages/playwright-core/src/server/injected/consoleApi.ts:91]: That's in CDT console, I don't think the expectation is to see it in the clipboard.

Comment (pavelfeldman) [reply packages/playwright-core/src/server/injected/consoleApi.ts:91]: It is multi-line and I need to console.log to preserve \n there.
